backend/src/main/java/com/redhat/cloud/notifications/routers/internal/InternalResource.java

```diff
@@ -469,4 +481,29 @@ public Response getDailyDigestTimePreference(@PathParam("orgId") String orgId) {
             return Response.status(NOT_FOUND).build();
         }
     }
+
+    /**
+     * Sends a daily digest command to the engine via a REST request.
+     * @param triggerDailyDigestRequestDto the settings of the digest.
+     */
+    @Consumes(APPLICATION_JSON)
+    @POST
+    @Path("/daily-digest")
```

Comment:
I wonder if we should prevent collisions with future internal endpoints related to the daily digest:
```suggestion
    @Path("/daily-digest/trigger")
```

Reply:
Changed!